### PR TITLE
update nitrogfx

### PR DESCRIPTION
**Pull request by red031000**

### CODE REVIEW

File: tools/nitrogfx/gfx.c [hunk 1]
```diff
@@ -719,9 +719,11 @@ void ReadNtrPalette(char *path, struct Palette *palette, int bitdepth, int palIn
     bitdepth = bitdepth ? bitdepth : palette->bitDepth;
 
     size_t paletteSize = (paletteHeader[0x10]) | (paletteHeader[0x11] << 8) | (paletteHeader[0x12] << 16) | (paletteHeader[0x13] << 24);
-    if (palIndex == 0) {
+    if (palIndex == 0)
```
Comment (PikalaxALT) [inline tools/nitrogfx/gfx.c:722]: Revert to 1TBS

File: tools/nitrogfx/main.c [hunk 2]
```diff
@@ -450,31 +450,31 @@ void HandlePngToNtrCommand(char *inputPath, char *outputPath, int argc, char **a
...
+            if (!ParseNumber(argv[i], NULL, 10, &options.colsPerChunk))
+                FATAL_ERROR("Failed to parse columns per chunk.\n");
 
-            if (options.metatileWidth < 1)
-                FATAL_ERROR("metatile width must be positive.\n");
+            if (options.colsPerChunk < 1)
+                FATAL_ERROR("columns per chunk must be positive.\n");
         }
-        else if (strcmp(option, "-mheight") == 0)
+        else if (strcmp(option, "-mheight") == 0 || strcmp(option, "-rpc") == 0)
```
Comment (PikalaxALT) [inline tools/nitrogfx/main.c:466]: Use 1TBS here

**Review by PikalaxALT — CHANGES_REQUESTED** [review]

File: tools/nitrogfx/gfx.c (hunk 1 above)
Comment (red031000) [reply tools/nitrogfx/gfx.c:722]: 1tbs?

Comment (red031000) [reply tools/nitrogfx/gfx.c:722]: have to disagree with this, as nitrogfx is mainly in Allman style (although somewhat inconsistently), until clang-format is merged (still on my todo list) and a linter is properly setup I am going to label this as out of scope, once those two are done I will ensure that nitrogfx complies for the future
(basically there are a few critical bug fixes here and a new feature among the renamings, and I want to get these in asap as the bug prevents the entire repo from building under certain systems such as cygwin)

Comment (red031000) [reply tools/nitrogfx/gfx.c:722]: main.c line 175, cast to unsigned char

Comment (red031000) [reply tools/nitrogfx/gfx.c:722]: there also is main.c line 291 correction of width to palindex

File: tools/nitrogfx/main.c [hunk 3]
```diff
@@ -381,31 +381,31 @@ void HandlePngToGbaCommand(char *inputPath, char *outputPath, int argc, char **a
             if (options.numTiles < 1)
                 FATAL_ERROR("Number of tiles must be positive.\n");
         }
-        else if (strcmp(option, "-mwidth") == 0)
+        else if (strcmp(option, "-mwidth") == 0 || strcmp(option, "-cpc") == 0)
         {
             if (i + 1 >= argc)
-                FATAL_ERROR("No metatile width value following \"-mwidth\".\n");
+                FATAL_ERROR("No columns per chunk value following \"%s\".\n", option);
```
Comment (adrienntindall) [inline tools/nitrogfx/main.c:387]: As an uninformed normie, "chunks" doesn't inherently make sense to me, which would be fine if it weren't for the fact that we could get people asking about what this error message means. Maybe I'm wrong and it makes sense tho

**Review by adrienntindall — CHANGES_REQUESTED** [review]

File: tools/nitrogfx/main.c (hunk 3 above)
Comment (red031000) [reply tools/nitrogfx/main.c:387]: a chunk is a collection of tiles
columns per chunk is how many columns of tiles there are per chunk
this used to be metatile but that was *more* confusing, cause metatile means... nothing really

Comment (adrienntindall) [reply tools/nitrogfx/main.c:387]: nitro studio documentation calls them cells and multicells, that work?

Comment (red031000) [reply tools/nitrogfx/main.c:387]: cells are another thing, don't want the same term for two different separate things

Comment (adrienntindall) [reply tools/nitrogfx/main.c:387]: eh can't think of anything better so it's prolly fine

**Review by adrienntindall — DISMISSED** [review]

**Comment by red031000** [comment]
https://media.discordapp.net/attachments/855279572651868170/1216412508986802276/image.png?ex=66004b6d&is=65edd66d&hm=178c0644e6a580fdbae8b368f4f5af8a1de4a2ff738dd255acab2728f1e6697b& fixes this pretty critical bug

**Comment by red031000** [comment]
(yes I know the link will die after a while, it's just temporary)

**Comment by PikalaxALT** [comment]
imma be a stickler for reverting the brace style changes

**Comment by red031000** [comment]
I've changed the two marked instances to 1tbs, however any other brace changes are out of scope for this PR

**Review by PikalaxALT — APPROVED** [review]

**Pull request merged**